src/main/java/card/Main.java

```diff
@@ -0,0 +1,20 @@
+package main.java.card;
+
+import main.java.card.validation.ValidateCard;
+import main.java.card.issuer.CardIssuers;
+
+public class Main {
+    public static void main(String[] args) {
+        CardIssuers.prepareCardIssuersMap();
+
+        ValidateCard.ResultTypeEnum validationResult = ValidateCard.performValidation(args);
```

Comment:
Prepare CreditCardInformation object before perform validation, the validation method should not be concerned with retrieving data

```suggestion
CardInputArgumentsControl inputControl = new CardInputArgumentsControl(inputArgs, 2);
ValidateCard.ResultTypeEnum validationResult = ValidateCard.performValidation(inputControl);
```